src/store/store.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable padded-blocks */
```

Comment:
Do you still need this? If so, can you disable just a single line, or perhaps move this before the block and then re-enable it after the block?